boards/native/native_sim/doc/index.rst

```diff
@@ -504,9 +504,12 @@ The following peripherals are currently provided with this board:
   enabled with :kconfig:option:`CONFIG_CAN_NATIVE_LINUX` and configured with the device tree binding
   :dtcompatible:`zephyr,native-linux-can`.
 
-  It is possible to specify which CAN interface will be used by the app using the ``--can-if``
-  command-line option. This option overrides **every** Linux SocketCAN driver instance to use the specified
-  interface.
+  By default, the native simulator expects a SocketCAN network device called ``zcan0``. It is
+  possible to specify which SocketCAN network device will be used by the app by using the
+  ``--can-if`` command-line option. This option overrides **every** Linux SocketCAN driver instance
+  to use the specified interface.
+
+  SocketCAN support can be enabled by using the :ref:`snippet-socketcan-native-sim`.
```

Comment:
```suggestion
  The default CAN controller in the native simulator is the loopback controller. SocketCAN support can be enabled by using the :ref:`snippet-socketcan-native-sim`.
```

And perhaps push this line a bit up to describe that loopback is the default and can be overridden with a host controller